spec/features/post_show_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'rails_helper'
+RSpec.describe 'post#how', type: :feature do
+  before(:each) do
+    @user = User.create(name: 'Amare', photo: 'My-photo', bio: 'Software Engineer')
+
+    @user2 = User.create(name: 'Solomon', bio: 'Elec Engineer',
+                         photo: 'sol-pic')
+
+    @post = Post.create(
+      title: 'Blog App dev',
+      text: 'Class association in detail there',
+      author_id: @user.id
+    )
+
+    Comment.create(text: 'Blog in rails is challenging', author_id: @user2.id, post_id: @post.id)
+    Comment.create(text: 'Yet you will adapt to it Sol', author_id: @user.id, post_id: @post.id)
+    Comment.create(text: 'Thanks', author_id: @user2.id, post_id: @post.id)
+    Like.create(post_id: @post.id, author_id: @user2.id)
+  end
+
+  describe 'post#show test group' do
+    it 'displays the post title' do
+      visit user_posts_path(@user, @post.id)
+      expect(page).to have_content('Class association in detail there')
```

Comment:
- [  OPTIONAL ] Great job working on all the tests but here I noticed you have a small issue this test failed because you are testing the title but you are putting the content so kindly just put `Blog App dev` instead of `Class association in detail there` to make the test pass as the others 💪🏻♻🔥💯💯💯

Reply:
Ok . Thank you @Reem-lab for giving me hints. Actually it works from my side. Perhaps my tests passed due to another bug from my side.